### PR TITLE
NATS SDK compatibility

### PR DESCRIPTION
**Pull request by OrMemphis**

### CODE REVIEW

File: server/memphis_helper.go [hunk 1]
```diff
@@ -503,58 +503,65 @@ func (s *Server) GetMessages(station models.Station, messagesToFetch int) ([]mod
 		messagesToFetch = int(totalMessages)
 	}
 
-	msgs, err := s.memphisGetMsgs(stationName.Intern()+".final",
-		stationName.Intern(),
+	msgs, err := s.memphisGetMsgs(stationName.Intern(),
```
Comment (idanasulinmemphis) [inline server/memphis_helper.go:506]: Why have you deleted it?

File: ui_static_files/build/asset-manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "files": {
```
Comment (idanasulinmemphis) [inline ui_static_files/build/asset-manifest.json:2]: remove all static files from the PR

File: server/memphis_helper.go (hunk 1 above)
Comment (OrMemphis) [reply server/memphis_helper.go:506]: Apparently, there was no use of that parameter inside the function, just cleaned it.

File: server/memphis_handlers_stations.go [hunk 3]
```diff
@@ -96,12 +96,24 @@ func validateReplicas(replicas int) error {
...
 	stationName, err := StationNameFromStr(station.Name)
 	if err != nil {
 		return err
 	}
-	err = s.RemoveStream(stationName.Intern())
+
+	removeFunc := nonNativeRemoveStreamFunc
+	if removeFunc == nil {
+		if !station.IsNative {
+			s.Fatalf("non native station uses native remove stream function")
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_stations.go:108]: please change to error and return the user an error

File: server/memphis_handlers_connections.go [hunk 4]
```diff
@@ -111,10 +134,10 @@ func handleConnectMessage(client *client) error {
 			Value: sdkName,
 		}
 		analyticsParams := []analytics.EventParam{param}
-		analytics.SendEventWithParams(user.Username, analyticsParams, "user-connect-sdk")
+		analytics.SendEventWithParams(username, analyticsParams, "user-connect-sdk")
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_connections.go:137]: please send new event user-connect-nats-sdk when this is non native connection

**Pull request merged**